confgenerator/testdata/valid/linux/metrics-no_conf/input.yaml

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 metrics:
-  service:
-    pipelines:
-      default_pipeline:
-        receivers: []
-        exporters: []
+  processors:
+    metrics_filter:
+      type: exclude_metrics
+      metrics_pattern:
+        - agent.googleapis.com/*
```

Comment:
Is this a good opportunity to [rename](https://github.com/GoogleCloudPlatform/ops-agent/pull/111#discussion_r660809764) this test case to `all_metrics_disabled` or `no_metrics` or something? Ditto for the Windows case.

Reply:
I'm trying to make the minimal change possible. Later.